components/teams-service/storage/postgres/postgres.go

```diff
@@ -212,7 +212,9 @@ func (p *postgres) GetTeams(ctx context.Context) ([]storage.Team, error) {
 	var teams []storage.Team
 	// TODO eventually these should be ordered
```

Comment:
Drive-by: is this still accurate? Do we have a card for it?

Reply:
Don't think our queries are sorted. Not sure if we have a card or not.